apps/site/app/[locale]/download/archive/[version]/page.tsx

```diff
@@ -0,0 +1,77 @@
+import { notFound } from 'next/navigation';
+import type { FC } from 'react';
+
+import { ENABLE_STATIC_EXPORT } from '#site/next.constants.mjs';
+import { ARCHIVE_DYNAMIC_ROUTES } from '#site/next.dynamic.constants.mjs';
+import * as basePage from '#site/next.dynamic.page.mjs';
+import { defaultLocale } from '#site/next.locales.mjs';
+
+type DynamicStaticPaths = { path: Array<string>; locale: string };
```

Comment:
```suggestion
type DynamicStaticPaths = { version: string; locale: string };
```

Read Next.js's App Router 👀 docs it'd make you understand how this file should be structured 🙇

Reply:
The params here are based on your path params FYI.